src/util/hass-util.html

```diff
@@ -63,12 +65,23 @@
   Object.keys(attributes).forEach(function (key) {
     customEl[key] = attributes[key];
   });
+  customEl.isVisible = true;
```

Comment:
I'm looking for a way to cancel the effect of a previous call to resetDynamicContent.